### PR TITLE
Josh

### PR DESCRIPTION
**Pull request by josh1den**

### CODE REVIEW

**Comment by josh1den** [comment]
@aliceafriedman -- can you show me how to merge this file with every change *except* for the line: TAKE ALL NEW COMMITS EXCEPT THIS ONE

Gonna keep trying to figure it out. No rush!

File: hw_1/iden_data624_hw1.Rmd [hunk 1]
```diff
@@ -55,10 +55,9 @@ grid.arrange(p1, p2, p3, nrow=3)
 #### b) What is the frequency of each series? Hint: apply the `frequency()` function.
 
 ```{r}
-string = 'data = %d times per year.\n'
-cat('gold', sprintf(string, frequency(gold)))
-cat('wool', sprintf(string, frequency(woolyrnq)))
-cat('gas', sprintf(string, frequency(gas)))
+sprintf('gold frequency = %d', frequency(gold))
+sprintf('wool frequency = %d', frequency(woolyrnq))
+sprintf('gas frequency = %d', frequency(gas))
```
Comment (josh1den) [inline hw_1/iden_data624_hw1.Rmd:60]: exclude

File: hw_1/iden_data624_hw1.Rmd [hunk 2]
```diff
@@ -59,6 +59,7 @@ sprintf('gold frequency = %d', frequency(gold))
 sprintf('wool frequency = %d', frequency(woolyrnq))
 sprintf('gas frequency = %d', frequency(gas))
 ```
+TAKE ALL NEW COMMITS EXCEPT THIS ONE
 
 Gold data is yearly, Wool data is quarterly, and Gas data is monthly.
```
Comment (josh1den) [inline hw_1/iden_data624_hw1.Rmd:64]: exclude

File: hw_1/iden_data624_hw1.Rmd (hunk 1 above)
Comment (aliceafriedman) [reply hw_1/iden_data624_hw1.Rmd:60]: Sorry it's not a super automated thing -- basically you have to have someone review and say, please revise and rollback this commit! 

OR if there's a conflict (which is when there are *competing* commits you can go in line by line and resolve the conflict.

**Review by aliceafriedman — CHANGES_REQUESTED** [review]
So basically you can "request changes' there's a way to "cherry pick" in the command line also...

**Pull request closed without merging**